packages/mobile/src/home/WalletHome.test.tsx

```diff
@@ -76,16 +89,31 @@ jest.mock('src/api/slice', () => ({
 }))
 
 describe('WalletHome', () => {
-  beforeAll(() => {
+  const mockFetch = fetch as FetchMock
+
+  beforeEach(() => {
     jest.useFakeTimers()
+    jest.clearAllMocks()
+    mockFetch.mockResponse(
+      JSON.stringify({
+        data: {
+          tokenTransactionsV2: {
+            transactions: [],
+          },
+        },
+      })
+    )
   })
 
-  afterAll(() => {
+  afterEach(() => {
```

Comment:
unrelated to this PR but i think we are setting the `useFakeTimers` as the default behaviour for jest [here](https://github.com/valora-inc/wallet/blob/deae905ea9e52f2764cc2cc90efb45c52e3cef63/packages/mobile/jest_setup.ts#L4) so i think we don't need this in the `beforeEach` and `afterEach`?

Reply:
True, just in case I will keep the `jest.useFakeTimers()` because we haven't been consistent about it and might be setting real timers back in other tests, to make sure we are using fake ones in this PR.